### PR TITLE
Add BladeGemm ROCm support

### PR DESCRIPTION
**Pull request by lfengad**

### CODE REVIEW

File: scripts/python/tao_build.py [hunk 1]
```diff
@@ -389,6 +391,11 @@ def _write(line, cmd="build"):
                 _action_env("BUILD_WITH_MKLDNN", "1")
             if args.aarch64:
                 _action_env("BUILD_WITH_AARCH64", "1")
+        else:
+            if args.platform_alibaba and args.blade_gemm:
```
Comment (fortianyou) [inline scripts/python/tao_build.py:395]: Nit: Are these configurations related to `args.platform_alibaba` capable to be moved to the internal repo?

**Comment by fortianyou** [comment]
Please add some descriptions for the pull request.

File: scripts/python/tao_build.py (hunk 1 above)
Comment (lfengad) [reply scripts/python/tao_build.py:395]: Currently the two version of build.py  are both with these configurations

File: scripts/python/tao_build.py [hunk 2]
```diff
@@ -389,6 +391,11 @@ def _write(line, cmd="build"):
                 _action_env("BUILD_WITH_MKLDNN", "1")
             if args.aarch64:
                 _action_env("BUILD_WITH_AARCH64", "1")
+        else:
```
Comment (minminsun) [inline scripts/python/tao_build.py:394]:
```suggestion
        elif args.dcu or args.rocm:
```

**Review by minminsun — APPROVED** [review]
LGTM

**Pull request merged**